core/chains/evm/log/integration_test.go

```diff
@@ -1306,7 +1174,7 @@ func TestBroadcaster_ReceivesAllLogsWhenResubscribing(t *testing.T) {
 			blockHeight1: 0,
 			batch1:       []uint{1, 2},
 
-			blockHeight2:     2,
+			blockHeight2:     3,
```

Comment:
This test is still broken. Every other case was fixed by an off-by-one adjustment, which makes sense based on the change to `SimulateIncomingHeads`/`Blocks.Slice` from an inclusive to exclusive end. :thinking: I'm hoping it was depending on incorrect behavior, like sending block 2 twice :crossed_fingers:

Reply:
Fixed! It was an off-by-two error :facepalm: